webapp/channels/src/components/pricing_modal/content.tsx

```diff
@@ -311,84 +326,94 @@ function Content(props: ContentProps) {
                 />
             </Modal.Header>
             <Modal.Body>
-                <div className='pricing-options-container'>
-                    <div className='alert-option-container'>
-                        <div className='alert-option'>
-                            <span>{formatMessage({id: 'pricing_modal.lookingToSelfHost', defaultMessage: 'Looking to self-host?'})}</span>
-                            <ExternalLink
-                                onClick={() =>
-                                    trackEvent(
-                                        TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
-                                        'click_looking_to_self_host',
-                                    )
-                                }
-                                href={CloudLinks.DEPLOYMENT_OPTIONS}
-                                location='pricing_modal_content'
-                            >{formatMessage({id: 'pricing_modal.reviewDeploymentOptions', defaultMessage: 'Review deployment options'})}</ExternalLink>
+                {!enabled && (
+                    <div className='pricing-options-container'>
+                        <div className='alert-option-container'>
+                            <div className='alert-option'>
+                                <span>{formatMessage({id: 'pricing_modal.lookingToSelfHost', defaultMessage: 'Looking to self-host?'})}</span>
+                                <ExternalLink
+                                    onClick={() =>
+                                        trackEvent(
+                                            TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
+                                            'click_looking_to_self_host',
+                                        )
+                                    }
+                                    href={CloudLinks.DEPLOYMENT_OPTIONS}
+                                    location='pricing_modal_content'
+                                >{formatMessage({id: 'pricing_modal.reviewDeploymentOptions', defaultMessage: 'Review deployment options'})}</ExternalLink>
+                            </div>
                         </div>
                     </div>
-                </div>
+                )}
+
+                <div
+                    className='PricingModal__body'
+                    style={{marginTop: enabled ? '74px' : ''}}
```

Comment:
nit: Use classNames for this and move styling into the scss file so all styles are in one place.

Reply:
3/5 since this is temporary, we can have it like this and when we're removing the feature flag, everything will be back in the scss files